services/state/token/account.proto

```diff
@@ -0,0 +1,176 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "basic_types.proto";
+import "crypto_approve_allowance.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.token">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * First-draft representation of a Hedera Token Service account entity in the network Merkle tree.
+ *
+ * As with all network entities, an account has a unique entity number, which is usually given along
+ * with the network's shard and realm in the form of a shard.realm.number id.
+ */
+
+message Account {
+    /**
+     * The unique entity number of the account.
```

Comment:
It might also be good to document that although we have an int64, only non-negative values are permitted.

Reply:
Later we should debate whether the shard and realm should be included here or not. Because I'm not sure if we need them to be part of the object for state proofs or not.

Reply:
If only non-negative numbers are permitted, should we make it a `uint32` or `uint64` and deal with it on the backend?

Reply:
Agreed @rbair23 . I think shard and realm can be common for a node in state or somewhere in one place.